templates/accounts/email_reset_email.html

```diff
@@ -11,6 +11,7 @@
 
```

Comment:
this is an email template, but is a .html file. Should maybe be changed if all the others are .txt?

Reply:
Addressed in 51479b1aa744a0f7dcae2f6b368b85487a5b2f00